Makefile

```diff
@@ -14,7 +14,7 @@
 
 # Image repo/tag to use all building/pushing image targets
 DOCKER_REGISTRY ?= ghcr.io/kubestellar/kubestellar
-IMAGE_TAG ?= 0.20.0-alpha.1
+IMAGE_TAG ?= 0.20.0-rc1
```

Comment:
Why does this PR even care about this `make` variable? Isn't #1779 the only one that should care?

Reply:
You are right, it should not care. This should be addressed by https://github.com/kubestellar/kubestellar/pull/1779 so I will just revert this to `0.20.0-alpha.1`